### PR TITLE
migrate_options_shared: some enhancements

### PR DESCRIPTION
**Pull request by Yingshun**
1. Ping_vm is unstable. It's not a mandatory checkpoint for
    migration cases, so we decide to remove it.
2. Update to access the migrated vm via console.

Signed-off-by: Yingshun Cui <yicui@redhat.com>
**Test result:**
```
# avocado run --vt-type libvirt virsh.migrate_options_shared.positive_test.vtpm_emulator.with_postcopy
WARNING:root:No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
WARNING:root:No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : ecd8c846d443b88f2ceccb498f9fa47d123d146c
JOB LOG    : /root/avocado/job-results/job-2021-03-22T03.22-ecd8c84/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_options_shared.positive_test.vtpm_emulator.with_postcopy: PASS (260.74 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 263.33 s

```

### CODE REVIEW

**Comment by Yingshun** [comment]
@dzhengfy Please help review this PR? Thanks!

**Review by smitterl — APPROVED** [review]

**Comment by chunfuwen** [comment]
@kylazhang 
@dzhengfy 
I have a little concerning here to remove ping_vm().
Previously I had one PR:https://github.com/autotest/tp-libvirt/pull/2844 ,where reviewers don't agree that ping_vm can be removed due to stability,and the PR was closed as invalid.

**Comment by Yingshun** [comment]
> @kylazhang
> @dzhengfy
> I have a little concerning here to remove ping_vm().
> Previously I had one PR:#2844 ,where reviewers don't agree that ping_vm can be removed due to stability,and the PR was closed as invalid.

I remember you said ping vm does not working in your case, even if you test that case manually, if so, I think it may be another problem. Or you can check if vm reboot works for you.

Something wrong was going on with the default network sometimes, the guest couldn't really connect for NAT setup. It makes migration job unstable, so we decide to remove it. For details please check the comment on https://github.com/autotest/tp-libvirt/pull/3218#discussion_r598470137

File: libvirt/tests/src/migration/migrate_options_shared.py [hunk 1]
```diff
@@ -1555,23 +1556,13 @@ def cancel_bg_migration():
...
-
             if cmd_in_vm_after_migration:
-                remote_session = remote.wait_for_login('ssh', server_ip, '22',
-                                                       server_user, server_pwd,
-                                                       r"[\#\$]\s*$")
-                vm_ip = vm.get_address(session=remote_session)
-                remote_session.close()
+                vm.connect_uri = dest_uri
+                vm_session_after_mig = vm.wait_for_serial_login(timeout=240)
+                vm_session_after_mig.cmd(cmd_in_vm_after_migration)
```
Comment (dzhengfy) [inline libvirt/tests/src/migration/migrate_options_shared.py:1562]: I think we still need to check the command execution results like old codes, right?

**Review by dzhengfy — APPROVED** [review]

**Comment by dzhengfy** [comment]
@fangge1212 Could you help review it?

File: libvirt/tests/src/migration/migrate_options_shared.py (hunk 1 above)
Comment (Yingshun) [reply libvirt/tests/src/migration/migrate_options_shared.py:1562]: It'll raise an exception by default if the command fails, so I think we don't need it.

Comment (dzhengfy) [reply libvirt/tests/src/migration/migrate_options_shared.py:1562]: OK. Agree because it is only a common check, not for a very specific checkout point.

**Review by dzhengfy — APPROVED** [review]
LGTM

**Review by fangge1212 — APPROVED** [review]

**Pull request merged**